docker/Dockerfile

```diff
@@ -79,7 +79,7 @@ FROM system-dependencies AS system-dev-dependencies
 # Install development/install-time OS dependencies
 # hadolint ignore=DL3008
 RUN apt-get update && \
-    apt-get install --no-install-recommends -y build-essential libssl-dev pkg-config libldap-dev libsasl2-dev libmariadb-dev nodejs npm && \
+    apt-get install --no-install-recommends -y build-essential libssl-dev pkg-config libldap-dev libsasl2-dev libmariadb-dev nodejs npm mariadb-client && \
```

Comment:
Just confirming that this is a dev dependency only? If it's needed in the prod images it should be added at line 51 instead.

Reply:
This should only need to be a dev dependency since it's just required to run the parallel unit tests